### PR TITLE
feat(YouTube/Seekbar components): Add info to `Append timestamp information` summary about tapping the appended information

### PR DESCRIPTION
**Pull request by KobeW50**
Resolves https://github.com/inotia00/ReVanced_Extended/issues/2459

### CODE REVIEW

File: src/main/resources/youtube/settings/host/values/strings.xml [hunk 1]
```diff
@@ -1067,7 +1067,8 @@ Tap and hold to undo."</string>
     <string name="revanced_preference_screen_seekbar_summary">Customize the seekbar components.</string>
 
     <string name="revanced_append_time_stamp_information_title">Append timestamp information</string>
-    <string name="revanced_append_time_stamp_information_summary_on">Information is appended to the timestamp.</string>
+    <string name="revanced_append_time_stamp_information_summary_on">"Information is appended to the timestamp.
+Tap to configure the video quality or playback speed. Tap and hold to toggle the appended information type."</string>
```
Comment (KobeW50) [inline src/main/resources/youtube/settings/host/values/strings.xml]: I wasn't sure if `Tap to configure the video quality or playback speed.` should be in the summaries of `revanced_append_time_stamp_information_type_summary` or if they should be together with the information about tapping-and-holding.

**Comment by Francesco146** [comment]
if it were possible, i'd extend the pr goal by adding to the string `revanced_hide_navigation_bar_summary_on` that, if activated, it may be impossible to reach the settings, and thus the deactivation of the toggle itself, through the usual path (i don't know if there are others, but they're certainly more intricate). thus resorting to a total reset of the settings if the user does not already have a backup

**Comment by KobeW50** [comment]
Good idea.
I was able to find this method of accessing the settings when the navigation bar is hidden, but I am not exactly sure how to describe these steps in text (for the summary) or if there is a better/simpler method


https://github.com/user-attachments/assets/cd369cfc-2007-4295-9d67-b3887fd58ae1

File: src/main/resources/youtube/settings/host/values/strings.xml (hunk 1 above)
Comment (inotia00) [reply src/main/resources/youtube/settings/host/values/strings.xml]: I think it will be fine

**Comment by Kuddus73** [comment]
> I am not exactly sure how to describe these steps in text (for the summary)

"Please use the following path to access the settings:
Home tab → Explore button → Trending → 3 vertical dots → Settings"

**Comment by Francesco146** [comment]
although this is correct, Explore i think it's hidden by default. anyway, it would be enough to add a "It may not be possible to reach the settings"

**Comment by KobeW50** [comment]
The category bar in the home feed is shown by default.

Also, kinda unrelated, but I realized that another way to enter the settings when the navigation bar is hidden is by enabling the `Replace Create button` toggle.

Anyways, I think it is fine to include the Explore button path while also mentioning that it may be impossible to access the settings.

One issue however, is that unlike `revanced_enable_wide_search_bar_in_you_tab_summary`, the summary for hiding the navigation bar is different for when it is `On` and `Off`. It seems a bit misplaced to have info about accessing the settings in the summary of the `Off` state, but displaying the information only when in the `On` state will not adequately inform users of this issue.

I think it is best to place the warning about accessing the settings in a popup dialog, though I understand that this is an experimental flag and I don't want to request this of @inotia00 (since I can't implement this myself).

**Comment by inotia00** [comment]
The `Enable wide search bar in You tab` setting should always show a warning, but hiding the navigation bar does not

If a user spoofs the app version or patches to YouTube 18.33.40 or earlier, they will still be able to access the settings even if the navigation bar is hidden, as the avatar button is located in the toolbar

**Pull request merged**